cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -330,7 +330,9 @@ def _extract_relationship_target_triples(self):
     def _extract_labels(self):
         for chunk in self._chunk(list(self._labels), description="Extracting labels"):
             label_iterator = self._client.labels.retrieve(external_id=list(chunk), ignore_unknown_ids=True)
-            yield from LabelsExtractor(label_iterator, **self._extractor_args).extract()
+            extractor = LabelsExtractor(label_iterator, **self._extractor_args)
+            extractor.lookup_dataset_external_id = self._lookup_dataset
+            yield from extractor.extract()
```

Comment:
Bug Labels did not set external ID of dataSet if externalId was passed in as the identifier.